### PR TITLE
[fix bug 1119551] Change fields on admin userprofile list

### PR DESCRIPTION
**Pull request by hoosteeno**
screen shot 2015-07-29 at 5 24 41 pm

### CODE REVIEW

**Comment by jwhitlock** [comment]
This code is going to change radically with PR #3352 - might need to wait a bit to redo this change.

File: kuma/users/admin.py [hunk 1]
```diff
@@ -60,6 +51,25 @@ def related_user(self, obj):
...
+        return ('<a href="%(link)s"><strong>%(count)s</strong></a>' %
+                {'link': link, 'count': count})
+    revisions.allow_tags = True
+
+    def websites(self, obj):
+        """The 'website' element in the 'websites' hash from the misc field"""
+        if obj.websites:
+            link = obj.websites.get('website')
+            return ('<a href="%(link)s"><strong>%(link)s</strong></a>' %
+                   {'link': link, 'link': link})
```
Comment (hoosteeno) [inline kuma/users/admin.py]: Ha, named string interpolation where all the names are the same. Will revisit in fixup.

**Comment by groovecoder** [comment]
Yeah :cry: ... the redo shouldn't be too different though. E.g., moving from `ProfileAdmin` to `UserAdmin`.

**Comment by groovecoder** [comment]
Closing until https://github.com/mozilla/kuma/pull/3352 is merged & pushed.

**Comment by hoosteeno** [comment]
@groovecoder is this code stable enough to reopen?

**Comment by groovecoder** [comment]
Yup, you can look at doing this again.

**Comment by hoosteeno** [comment]
https://github.com/mozilla/kuma/pull/3579

**Pull request closed without merging**